### DISCUSSION TITLE
Support for Windows

### DISCUSSION THREAD

**Discussion by hanshupe007** [ServiceNow/azimuth#599] [General]
Is Azimuth fully supported on Windows? I tried to follow the instructions, but not sure how to run the make command etc. on Windows?

**Comment by JosephMarinier** [comment]
Hi @hanshupe007! Thank you for your interest.
We have never tried to run Azimuth on Windows, but it should be relatively simple given it is containerized (Docker).
As for `make`, it is not as straightforward. I did a quick search on Google and I saw some ways to install GNU make on Windows, or other workarounds. That said, it is not necessary. You can probably look at the definitions of the `make` rules you want to run (in our `Makefile`) and run the commands directly. They should be fairly simple. You might need to adapt some of the syntax to the Windows shell, of course, or you could use bash (via the native Windows Subsystem for Linux) if that's more straightforward for you.
Hopefully, that gives you a couple of hints of where to start. I'm sure you can figure it out faster than us (not familiar with Windows). 😅 Let me know if you would need more assistance. I would be able to get a Windows machine and help you.
